### DISCUSSION TITLE
Vapourwave and nighcore effect

### DISCUSSION THREAD

**Discussion by TheShubhendra** [lavalink-devs/Lavalink#562] [Q&A] [answered]
How can I create nighcore and vapourwave effects using filters?

**Comment by topi314** [comment, marked as answer]
no idea what those certain effects are
but for general filter usage see https://github.com/freyacodes/Lavalink/blob/master/IMPLEMENTATION.md#using-filters

you can combine multiple filters together and create effects as you wish
